api/views/phones.py

```diff
@@ -442,14 +442,17 @@ def inbound_sms(request):
 
     inbound_contact = _get_inbound_contact(relay_number, inbound_from)
     if inbound_contact:
-        _check_and_update_contact(inbound_contact, "texts")
+        _check_and_update_contact(inbound_contact, "texts", relay_number)
 
     client = twilio_client()
     client.messages.create(
         from_=relay_number.number,
         body=f"[Relay 📲 {inbound_from}] {inbound_body}",
         to=real_phone.number,
     )
+    relay_number.remaining_texts -= 1
```

Comment:
TODO: (either here or somewhere else) add logic to:
1. check if `remaining_texts` need to be reset to the limit because we're in a new billing period since the customer's subscription started.
2. check if `remaining_texts` is 0 - don't forward the text and/or text the user that this is their last text for the period

Reply:
3. and if `relay_number.remaining_texts` is `None`, reset to limit. Same for other places where `null` is allowed